### DISCUSSION TITLE
Helm Chart package has unexpected ArtifactHub path segment for organization

### DISCUSSION THREAD

**Discussion by jfallows** [artifacthub/hub#3154] [General]
Zilla package lives in the Aklivity organization in ArtifactHub.

However, searching for Zilla package on ArtifactHub gives
 - https://artifacthub.io/packages/helm/zilla/zilla
instead of
 - https://artifacthub.io/packages/helm/aklivity/zilla

Is there an ArtifactHub organization setting that we can adjust to change the Aklivity organization path segment in the repository package URL from `zilla` to `aklivity` (but keeping the repository path segment as `zilla`) ?

Many thanks in advance.

**Comment by tegioz** [comment]
Hi @jfallows 👋

No, we don't use the organization name in the urls. The first `zilla` instance in the url corresponds to the repository name, whereas the second one refers to the package name (chart in this case). You could add a repository named `aklivity` that would contain the `zilla` chart, producing the url desired. Repositories cannot be renamed though, so you'd need to create a new one.

Hope this helps 🙂

**Reply by jfallows** [reply]
Thank you for the clarification, all is well then. 👍

**Reply by tegioz** [reply]
Cool, no worries 👍
